packages/pmml-editor/src/editor/components/EditorScorecard/molecules/CharacteristicsToolbar.tsx

```diff
@@ -74,7 +73,7 @@ export const CharacteristicsToolbar = (props: CharacteristicsToolbarProps) => {
                       data-testid="characteristics-toolbar__submit"
                       variant={ButtonVariant.control}
                       aria-label="filter button for filter input"
-                      onClick={() => onFilter(filter)}
+                      onClick={() => onFilter()}
```

Comment:
No need to create a new lambda expression here. You can use `onFilter` directly. The same applies for changes below.
```suggestion
                      onClick={onFilter}
```

Reply:
Corrected (in both places).